packages/web-components/src/components/rux-button/test/basic/button.vrt.spec.ts

```diff
@@ -1,16 +1,9 @@
 import { test, expect } from '../../../../../tests/utils/_astro-fixtures'
 
 test.describe('Button', () => {
-    test('has no visual regression @vrt @dark', async ({ page }) => {
-        await page.goto(`/src/components/rux-button/test/basic/index.html`)
-        await expect(page).toHaveScreenshot()
-    })
+    test.use({ component: 'rux-button' })
 
-    test('has no visual regression @vrt @light', async ({ page }) => {
-        await page.goto(`/src/components/rux-button/test/basic/index.html`)
-        await page.evaluate(() => {
-            document.body.classList.add('light-theme')
-        })
-        await expect(page).toHaveScreenshot()
+    test('has no visual regression @vrt', async ({ astroVRTPage }) => {
+        await expect(astroVRTPage).toHaveScreenshot()
```

Comment:
is this too much of an abstraction? the actual goto step is moved into the fixture. one downside I can see to this is it might be confusing if youre unaware and trying to figure out where the test setup is. 

but the big value add here is not having to write duplicate tests for dark and light theme. if we move goto to a fixture, we can automatically append the light theme class after load.

Reply:
another con--having to add another var if/when we expand into more detailed test dirs other than 'basic'.